README.md

```diff
@@ -25,3 +25,46 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
+
+[Download Free Trial](https://www.syncfusion.com/downloads/angular?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/angular?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples)
+
+[Documentation](https://angular.syncfusion.com/documentation/progress-button/getting-started?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples)
+
+[View Demos](https://angular.syncfusion.com/demos/progress-button/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/angular-components?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/angular-components?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples)
+
+[Online example](https://angular.syncfusion.com/demos/toolbar/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples)
+
+[Video Tutorials](https://www.syncfusion.com/tutorial-videos/angular/toolbar?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples)
+
+[Knowledge base](https://www.syncfusion.com/kb/angular-components?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples)
+
+## About Syncfusion Angular Components
+The Syncfusion's [Angular components](https://www.syncfusion.com/angular-ui-components?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples) library offers over 70 UI components to work with Angular server-side and client-side (Angular WebAssembly) projects seamlessly. In addition to Progress Button, we provide popular Angular Components such as [DataGrid](https://www.syncfusion.com/angular-components/angular-grid?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples),[Charts](https://www.syncfusion.com/angular-components/angular-charts?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples), [Scheduler](https://www.syncfusion.com/angular-components/angular-scheduler?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples), [Diagram](https://www.syncfusion.com/angular-components/angular-diagram?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples) and [Word Processor](https://www.syncfusion.com/angular-components/angular-word-processor?utm_source=github&utm_medium=listing&utm_campaign=angular-progress-buttons-github-samples).
```

Comment:
Modify the content like the below PR
https://github.com/SyncfusionExamples/getting-started-with-the-angular-toolbar-component/pull/3/files